### PR TITLE
70 filters for search results

### PR DESCRIPTION
**Pull request by jchristensonCHI**
Builds off of PR:"124 sort the search by"

### CODE REVIEW

**Review by claireboyd — COMMENTED** [review]
LTGM! added a few comments, but more out of curiosity then anything

**Review by claireboyd — APPROVED** [review]

File: civiclens/regulations/views.py [hunk 1]
```diff
@@ -19,10 +24,23 @@ def search_page(request):
...
+            comments_over_hundred = request.GET.get("comments_over_hundred")
+            category_rule = request.GET.get("rule")
+            category_proprosed_rule = request.GET.get("proposed_rule")
+            category_notice = request.GET.get("notice")
+            category_other = request.GET.get("other")
+            category_lst = [category_rule, category_proprosed_rule, category_notice, category_other]
+            
         if query:
             vector = (
                 SearchVector("title", weight="A")
```
[the remark this answers is not in the document]
Comment (jchristensonCHI) [reply civiclens/regulations/views.py:46]: I thought those categories would be weighted to high at weight C.

File: civiclens/regulations/views.py [hunk 2]
```diff
@@ -77,4 +113,4 @@ def document(request, doc_id):
 
 
 def comment(request):
```
[the remark this answers is not in the document]
Comment (jchristensonCHI) [reply civiclens/regulations/views.py:115]: I will do that in a future issue!

File: civiclens/regulations/views.py [hunk 3]
```diff
@@ -19,10 +24,23 @@ def search_page(request):
 
 
 def search_results(request):
+    today = timezone.now().date()
     context = {}
+    
     if request.method == "GET":
         query = request.GET.get("q", "")
-
+        sort_by = request.GET.get("sort_by", "most_relevant")
```
[the remark this answers is not in the document]
Comment (jchristensonCHI) [reply civiclens/regulations/views.py:32]: Thanks!

**Pull request merged**